gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/metastore/HiveMetaStoreBasedRegister.java

```diff
@@ -632,7 +632,7 @@ private void addOrAlterPartitionWithPullMode(IMetaStoreClient client, Table tabl
             onPartitionExist(client, table, partition, nativePartition, existedPartition);
           }
         }
-      } catch (TException e) {
+      } catch (NoSuchObjectException e) {
```

Comment:
is this relevant ?

Reply:
I think this is introduced by other three commit. Can you please change the changes specific to last two avro specific commits.